controllers/account/ec2.go

```diff
@@ -31,7 +31,6 @@ const T3INSTANCETYPE = "t3.micro"
 const T2INSTANCETYPE = "t2.micro"
 
 var sampleCIDR = "10.0.0.0/16"
-var sampleVPCID = ""
```

Comment:
There is a comment on line 143 that can be removed/updated to reflect this var being removed